enterprise/management/commands/compare_discovery_and_enterprise_catalogs.py

```diff
@@ -43,6 +45,13 @@ def handle(self, *args, **options):
                     )
                     continue
 
+                if catalog_query.content_filter.get('aggregation_key'):
```

Comment:
catalogs which explicitly request specific course keys we should skip modification of.

`aggregation_key__not` we will modify though.